### ISSUE TITLE
ownership and mode of authorized_keys

### ISSUE THREAD

**Issue by jflournoy** [ethack/docker-vpn#1]
I had a problem connecting to the container via ssh which was fixed by adding the following to the docker-entrypoint.sh:

```
#change ownership and mode
chown root /root/.ssh/authorized_keys
chmod 600 /root/.ssh/authorized_keys
```

**Comment by jflournoy** [comment]
I realized that, since this file may on a mounted volume, it might be better to not mount it to /root/.ssh but instead copy whatever file is either mounted or created and then change ownership and permissions.

**Comment by ethack** [comment]
Closed by #2
